### ISSUE TITLE
Daily search not finding episodes - Danishbits.org

### ISSUE THREAD

**Issue by mattail** [pymedusa/Medusa#3587]
### Before submitting your issue:

Enable debug logging in Medusa settings, reproduce the error (be sure to disable after the bug is fixed)

Branch: master
Commit: 7b9111eef5836a6846499ef257ffff3f1f35df8f
Version: v0.1.22
OS: Linux-3.10.102-x86_64-with-glibc2.2.5 / Synology

Daily-search does not find all the different episode releases on search provider Danishbits.org.
While browsing the torrent provider after I got home from work I noticed that a new episode of a show "Star-Trek: discovery" was released but was not picked up by the daily-search. So I forced a daily-search from within Medusa to see if woul be picked up but that was not the case. So I checked the logs and see that even there are other releases for the same episode from other groups then this one from the logs below is the only one found (Star.Trek.Discovery.S01E10.1080p.WEBRip.x264-CONVOY).

**Logs:**
```
PASTE LOGS HERE
2018-01-08 16:35:43 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [Danishbits] :: [7b9111e] Using cached parse result for Star.Trek.Discovery.S01E10.1080p.WEBRip.x264-CONVOY
2018-01-08 16:35:43 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [Danishbits] :: [7b9111e] 328711: Allowed, Preferred = [ 1080p HDTV, 1080p WEB-DL, 1080p BluRay ] [ None ] Found = [ 1080p WEB-DL ]
2018-01-08 16:35:43 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [Danishbits] :: [7b9111e] 328711: 'Star Trek: Discovery' S01E10 status is: 'WANTED'. Accepting result with quality '1080p WEB-DL'. Reason: Current status is 'WANTED'. Accepting result with quality '1080p WEB-DL'
2018-01-08 16:35:43 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [Danishbits] :: [7b9111e] 328711: Using cached results from Danishbits for series u'Star Trek: Discovery' episode S01E10
2018-01-08 16:35:43 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [Danishbits] :: [7b9111e] Picking the best result out of [u'Star.Trek.Discovery.S01E10.1080p.WEBRip.x264-CONVOY']
2018-01-08 16:35:43 INFO     SEARCHQUEUE-DAILY-SEARCH :: [Danishbits] :: [7b9111e] Quality of Star.Trek.Discovery.S01E10.1080p.WEBRip.x264-CONVOY is 1080p WEB-DL
2018-01-08 16:35:43 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [Danishbits] :: [7b9111e] 328711: Allowed, Preferred = [ 1080p HDTV, 1080p WEB-DL, 1080p BluRay ] [ None ] Found = [ 1080p WEB-DL ]
2018-01-08 16:35:43 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [Danishbits] :: [7b9111e] 328711: 'Star Trek: Discovery' S01E10 status is: 'WANTED'. Accepting result with quality '1080p WEB-DL'. Reason: Current status is 'WANTED'. Accepting result with quality '1080p WEB-DL'
2018-01-08 16:35:43 INFO     SEARCHQUEUE-DAILY-SEARCH :: [Danishbits] :: [7b9111e] Ignoring Star.Trek.Discovery.S01E10.1080p.WEBRip.x264-CONVOY based on required words filter: [u'NORDIC']
2018-01-08 16:35:43 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [Danishbits] :: [7b9111e] No result picked.
2018-01-08 16:35:43 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [Danishbits] :: [7b9111e] 328711: Formatting pattern: %SN - S%0SE%0E - %EN -> Star Trek Discovery - S01E10 - Despite Yourself
2018-01-08 16:35:43 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [Danishbits] :: [7b9111e] All found results for Star Trek Discovery - S01E10 - Despite Yourself were rejected.

**Comment by medariox** [comment]
Did you try to manual search the episode? You will probably have to add a scene exception. Enable debug logs and post the log results of the search for more help.

**Comment by mattail** [comment]
Episode is still not found either by manual or force search.
I have set quality to "HD1080p" and required word "NORDIC".
The name of the episode release it should catch is: Star.Trek.Discovery.S01E10.NORDiC.1080p.WEB-DL.DD5.1.H.264-DBRETAiL 

Manual search:
2018-01-08 18:58:16 INFO     SEARCHQUEUE-MANUAL-328711 :: [7b9111e] Using manual search providers
2018-01-08 18:58:16 INFO     SEARCHQUEUE-MANUAL-328711 :: [Danishbits] :: [7b9111e] Performing episode search for Star Trek: Discovery
2018-01-08 18:58:16 DEBUG    SEARCHQUEUE-MANUAL-328711 :: [Danishbits] :: [7b9111e] Search mode: Episode
2018-01-08 18:58:16 DEBUG    SEARCHQUEUE-MANUAL-328711 :: [Danishbits] :: [7b9111e] Search string: Star Trek: Discovery S01E10
2018-01-08 18:58:17 DEBUG    SEARCHQUEUE-MANUAL-328711 :: [Danishbits] :: [7b9111e] GET URL: https://danishbits.org/couchpotato.php?search=Star+Trek%3A+Discovery+S01E10&latest=false&user=**********&passkey=********** [Status: 200]
2018-01-08 18:58:17 DEBUG    SEARCHQUEUE-MANUAL-328711 :: [Danishbits] :: [7b9111e] User-Agent: Medusa/0.1.21 (Linux; 3.10.102; a787ce30-f0c3-11e7-8fe5-02422400d0ff)
2018-01-08 18:58:17 DEBUG    SEARCHQUEUE-MANUAL-328711 :: [Danishbits] :: [7b9111e] Found 0 unique items
2018-01-08 18:58:17 DEBUG    SEARCHQUEUE-MANUAL-328711 :: [Danishbits] :: [7b9111e] Found qualities: []
2018-01-08 18:58:17 DEBUG    SEARCHQUEUE-MANUAL-328711 :: [Danishbits] :: [7b9111e] No unknown qualities in results
2018-01-08 18:58:17 DEBUG    SEARCHQUEUE-MANUAL-328711 :: [7b9111e] 328711: Formatting pattern: %SN - S%0SE%0E - %EN -> Star Trek Discovery - S01E10 - Despite Yourself
2018-01-08 18:58:17 INFO     SEARCHQUEUE-MANUAL-328711 :: [7b9111e] Unable to find manual results for: Star Trek Discovery - S01E10 - Despite Yourself
2018-01-08 18:59:19 DEBUG    SEARCHQUEUE-MANUAL-328711 :: [7b9111e] 328711: Formatting pattern: %SN - S%0SE%0E - %EN -> Star Trek Discovery - S01E10 - Despite Yourself
2018-01-08 18:59:19 INFO     SEARCHQUEUE-MANUAL-328711 :: [7b9111e] Beginning manual search for: Star Trek Discovery - S01E10 - Despite Yourself
2018-01-08 18:59:19 INFO     SEARCHQUEUE-MANUAL-328711 :: [7b9111e] Updating exception_cache and exception_season_cache
2018-01-08 18:59:19 INFO     SEARCHQUEUE-MANUAL-328711 :: [7b9111e] Finished processing 2692 scene exceptions.
2018-01-08 18:59:19 INFO     SEARCHQUEUE-MANUAL-328711 :: [7b9111e] Building internal name cache for Star Trek: Discovery
2018-01-08 18:59:19 DEBUG    SEARCHQUEUE-MANUAL-328711 :: [7b9111e] Internal name cache for Star Trek: Discovery set to: star trek discovery

Force search:
018-01-08 19:01:37 DEBUG    SEARCHQUEUE-FORCED-328711 :: [7b9111e] 328711: Formatting pattern: %SN - S%0SE%0E - %EN -> Star Trek Discovery - S01E10 - Despite Yourself
2018-01-08 19:01:37 INFO     SEARCHQUEUE-FORCED-328711 :: [7b9111e] Beginning forced search for: Star Trek Discovery - S01E10 - Despite Yourself
2018-01-08 19:01:37 INFO     SEARCHQUEUE-FORCED-328711 :: [7b9111e] Updating exception_cache and exception_season_cache
2018-01-08 19:01:37 INFO     SEARCHQUEUE-FORCED-328711 :: [7b9111e] Finished processing 2692 scene exceptions.
2018-01-08 19:01:37 INFO     SEARCHQUEUE-FORCED-328711 :: [7b9111e] Building internal name cache for Star Trek: Discovery
2018-01-08 19:01:37 DEBUG    SEARCHQUEUE-FORCED-328711 :: [7b9111e] Internal name cache for Star Trek: Discovery set to: star trek discovery
2018-01-08 19:01:37 INFO     SEARCHQUEUE-FORCED-328711 :: [7b9111e] Using backlog search providers
2018-01-08 19:01:37 INFO     SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] Performing episode search for Star Trek: Discovery
2018-01-08 19:01:37 DEBUG    SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] Using cached parse result for Star.Trek.Discovery.S01E10.1080p.WEBRip.x264-CONVOY
2018-01-08 19:01:37 DEBUG    SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] 328711: Allowed, Preferred = [ 1080p HDTV, 1080p WEB-DL, 1080p BluRay ] [ None ] Found = [ 1080p WEB-DL ]
2018-01-08 19:01:37 DEBUG    SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] 328711: 'Star Trek: Discovery' S01E10 status is: 'WANTED'. Accepting result with quality '1080p WEB-DL'. Reason: Current status is 'WANTED'. Accepting result with quality '1080p WEB-DL'
2018-01-08 19:01:37 DEBUG    SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] 328711: Using cached results from Danishbits for series u'Star Trek: Discovery' episode S01E10
2018-01-08 19:01:37 DEBUG    SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] Using cached parse result for Star.Trek.Discovery.S01E10.WEBRip.x264-TBS
2018-01-08 19:01:37 DEBUG    SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] 328711: Allowed, Preferred = [ 1080p HDTV, 1080p WEB-DL, 1080p BluRay ] [ None ] Found = [ SDTV ]
2018-01-08 19:01:37 DEBUG    SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] 328711: Ignoring found result for 'Star Trek: Discovery' S01E10 with unwanted quality 'SDTV'
2018-01-08 19:01:37 DEBUG    SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] Ignoring Star.Trek.Discovery.S01E10.WEBRip.x264-TBS
2018-01-08 19:01:37 DEBUG    SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] Using cached parse result for Star.Trek.Discovery.S01E10.720p.WEBRip.x264-TBS
2018-01-08 19:01:37 DEBUG    SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] 328711: Allowed, Preferred = [ 1080p HDTV, 1080p WEB-DL, 1080p BluRay ] [ None ] Found = [ 720p WEB-DL ]
2018-01-08 19:01:37 DEBUG    SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] 328711: Ignoring found result for 'Star Trek: Discovery' S01E10 with unwanted quality '720p WEB-DL'
2018-01-08 19:01:37 DEBUG    SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] Ignoring Star.Trek.Discovery.S01E10.720p.WEBRip.x264-TBS
2018-01-08 19:01:38 DEBUG    SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] The highest quality of any match is 1080p WEB-DL
2018-01-08 19:01:38 DEBUG    SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] Picking the best result out of [u'Star.Trek.Discovery.S01E10.1080p.WEBRip.x264-CONVOY']
2018-01-08 19:01:38 INFO     SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] Quality of Star.Trek.Discovery.S01E10.1080p.WEBRip.x264-CONVOY is 1080p WEB-DL
2018-01-08 19:01:38 DEBUG    SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] 328711: Allowed, Preferred = [ 1080p HDTV, 1080p WEB-DL, 1080p BluRay ] [ None ] Found = [ 1080p WEB-DL ]
2018-01-08 19:01:38 DEBUG    SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] 328711: 'Star Trek: Discovery' S01E10 status is: 'WANTED'. Accepting result with quality '1080p WEB-DL'. Reason: Current status is 'WANTED'. Accepting result with quality '1080p WEB-DL'
2018-01-08 19:01:38 INFO     SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] Ignoring Star.Trek.Discovery.S01E10.1080p.WEBRip.x264-CONVOY based on required words filter: [u'NORDIC']
2018-01-08 19:01:38 DEBUG    SEARCHQUEUE-FORCED-328711 :: [Danishbits] :: [7b9111e] No result picked.
2018-01-08 19:01:38 DEBUG    SEARCHQUEUE-FORCED-328711 :: [7b9111e] 328711: Formatting pattern: %SN - S%0SE%0E - %EN -> Star Trek Discovery - S01E10 - Despite Yourself
2018-01-08 19:01:38 INFO     SEARCHQUEUE-FORCED-328711 :: [7b9111e] Unable to find forced results for: Star Trek Discovery - S01E10 - Despite Yourself

**Comment by medariox** [comment]
Add `Star Trek Discovery` as scene exception and try again.

**Comment by mattail** [comment]
I did enter Star Trek Discovery as scene exception saved changes, then made a manual search, nothing was found. Then i repeatedly pressed the force search button multiple times. The first result retuned 3 results but after a few more press 3 extra hits appeared where the one i wanted was amoung them. So I did a force daily seach again and the episode was picked.
I'm not sure if the scene exception did the trick or if it is because of some cache issue.
I will keep a closer look the next days and see if this happens with other tv-shows. 

I've just started using Medusa, coming from Sickrage where this ran smoothly. 

Thank you for your help medariox.

**Comment by medariox** [comment]
After you add the scene exception you have to force the search _once_ otherwise, you'll only see the cached results. The colon in `Star Trek: Discovery` apparently broke the search for Danishbits, maybe it's an unsupported char for the search (I can't verify as I'm not a member there).

**Comment by mattail** [comment]
Ok thanks. Will try same method if I see same issue with other shows.

**Comment by IguanaC64** [comment]
So here I am six years later because I just hit this exact issue with this exact same show, but I have some more input to add...

Both Jackett and Prowlarr were tested using Alpharatio, TLZ, and Tokyo Toshokan:
Jackett: I typed "Trek" and got results from all three, I typed "Trek:" and only got results on Tokyo Toshokan
Prowlarr: I typed "Trek" and got results from all three, I typed "Trek:" and only got results on Alpharatio and Tokyo Toshokan
Cardigann*: I typed "Trek" and got results from Alpharatio, I typed "Trek:" and got zero results

* Cardigann only had a connector for Alpharatio, so TLZ and Tokyo Toshokan were not tested.

Yes...making a scene exception and restarting Medusa fixed the issue for Star Trek Discovery, but it's apparent that colons are breaking searches for different software and different sites.  Is there a reason Medusa can't filter out colons automatically?  Would that significantly blow up searches?  Do we have to contact someone at XEM to add an automatic scene exception?  Should this be opened as a new issue or will this post be sufficient to get a response?
